apps/minecraftserver/minecraft_server.star

```diff
@@ -0,0 +1,78 @@
+"""
+Applet: Minecraft Server
+Summary: Minecraft Server Activity
+Description: View Minecraft Server Activity and icon.
+Author: Michael Blades
+"""
+
+load("render.star", "render")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("schema.star", "schema")
+load("cache.star", "cache")
+
+def main(config):
+    result_cached = cache.get("api_result")
```

Comment:
The cache is namespaced to this particular app, but not to individual users. This means all users will in this case use the same `"api_result"` cache key, which is probably not what you want. A simple fix would be to include the `server` config value in the cache key to make sure the result is unique to that particular config.

Reply:
@matslina These changes have been made thanks for the advice!!